internal/nodes/ingress_model_rel.go

```diff
@@ -116,6 +116,11 @@ var (
 		Type:                           lib.ServiceImport,
 		GetParentMultiClusterIngresses: ServiceImportToMultiClusterIng,
 	}
+	OAuthSamlConfig = GraphSchema{
+		Type:               "OAuthSamlConfig",
```

Comment:
Better to reuse `lib.OAuthSamlConfig` constant here.

Reply:
Done